_aggregations/metric/percentile.md

```diff
@@ -9,86 +9,381 @@ redirect_from:
 
 # Percentile aggregations
 
-Percentile is the percentage of the data that's at or below a certain threshold value.
+The `percentiles` aggregation estimates the value at a given percentile of a numeric field. This is useful for understanding distribution boundaries.
 
-The `percentile` metric is a multi-value metric aggregation that lets you find outliers in your data or figure out the distribution of your data.
+For example, a 95th percentile of `load_time` = `120ms` means 95% of values are less than or equal to 120ms.
 
-Like the `cardinality` metric, the `percentile` metric is also approximate.
+Similarly to the [`cardinality`]({{site.url}}{{site.baseurl}}/aggregations/metric/cardinality/) metric, the `percentile` metric is approximate.
 
-The following example calculates the percentile in relation to the `taxful_total_price` field:
+## Parameters
+
+The `percentiles` aggregation takes the following parameters.
+
+| Parameter                                | Data type        | Required/Optional | Description                                                                                                                 |
+| ---------------------------------------- | ---------------- | -------- | --------------------------------------------------------------------------------------------------------------------------- |
+| `field`                                  | String           | Required      | The numeric field used to compute percentiles on.                                                                                    |
+| `percents`                               | Array of doubles | Optional       | The list of percentiles returned in the response. Default is `[1, 5, 25, 50, 75, 95, 99]`.                                                 |
+| `keyed`                                  | Boolean          | Optional       | If set to `false` returns results as a array, otherwise returns results as JSON object. Default is `true` |
```

Comment:
```suggestion
| `keyed`                                  | Boolean          | Optional       | If set to `false`, returns results as an array. Otherwise, returns results as a JSON object. Default is `true`. |
```